components/secret-mgt/org.wso2.carbon.identity.secret.mgt.core/pom.xml

```diff
@@ -0,0 +1,232 @@
+                        </Import-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.identity.secret.mgt.core.internal,
+                            org.wso2.carbon.identity.secret.mgt.core.*; version="${project.version}",
+                        </Export-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+              <plugin>
+                  <groupId>org.apache.maven.plugins</groupId>
+                  <artifactId>maven-surefire-plugin</artifactId>
+                  <version>${maven.surefire.plugin.version}</version>
+                  <configuration>
+                      <suiteXmlFiles>
+                          <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
+                      </suiteXmlFiles>
+                  </configuration>
+              </plugin>
+              <plugin>
+                  <groupId>org.jacoco</groupId>
+                  <artifactId>jacoco-maven-plugin</artifactId>
+                  <version>${jacoco.version}</version>
+                  <executions>
+                      <execution>
+                          <id>default-prepare-agent</id>
+                          <goals>
+                              <goal>prepare-agent</goal>
+                          </goals>
+                      </execution>
+                      <execution>
+                          <id>default-prepare-agent-integration</id>
+                          <goals>
+                              <goal>prepare-agent-integration</goal>
+                          </goals>
+                      </execution>
+                      <execution>
+                          <id>default-report</id>
+                          <goals>
+                              <goal>report</goal>
+                          </goals>
+                      </execution>
+                      <execution>
+                          <id>default-report-integration</id>
+                          <goals>
+                              <goal>report-integration</goal>
+                          </goals>
+                      </execution>
+                      <execution>
+                          <id>default-check</id>
+                          <goals>
+                              <goal>check</goal>
+                          </goals>
+                          <configuration>
+                              <rules>
+                                  <rule implementation="org.jacoco.maven.RuleConfiguration">
+                                      <element>BUNDLE</element>
+                                      <limits>
+                                          <limit implementation="org.jacoco.report.check.Limit">
+                                              <counter>COMPLEXITY</counter>
+                                              <value>COVEREDRATIO</value>
+                                              <!--<minimum>0.60</minimum>-->
```

Comment:
As we have the test cases added, we can uncomment this line and have the test covered percentage.

Reply:
resolved

Reply:
we need to add the current test coverage percentage as the minimum value

Reply:
resolved